string name
---
object_identification/AttributeArray attributes

